action/protocol/rewarding/reward.go

```diff
@@ -123,12 +124,14 @@ func (p *Protocol) GrantBlockReward(
 		}
 		msg proto.Message = rewardLogs[0]
 	)
-	if !isZero(effectiveTip) {
-		rewardLogs = append(rewardLogs, &rewardingpb.RewardLog{
-			Type:   rewardingpb.RewardLog_PRIORITY_BONUS,
-			Addr:   rewardAddrStr,
-			Amount: effectiveTip.String(),
-		})
+	if featureCtx.EnableDynamicFeeTx {
+		if !isZero(effectiveTip) {
+			rewardLogs = append(rewardLogs, &rewardingpb.RewardLog{
+				Type:   rewardingpb.RewardLog_PRIORITY_BONUS,
+				Addr:   rewardAddrStr,
+				Amount: effectiveTip.String(),
+			})
+		}
```

Comment:
should always use `msg = &rewardingpb.RewardLogs` after `EnableDynamicFeeTx` flag